reactor-core/src/main/java/reactor/core/publisher/FluxPublish.java

```diff
@@ -120,15 +127,21 @@ public void subscribe(CoreSubscriber<? super T> actual) {
 				c = u;
 			}
 
+			inner.parent = c;
```

Comment:
looks like a behavior change. not about the impact but lets keep the old logic as it was

Reply:
Right. In the earlier stages this was necessary, but it should be only necessary in the previous logic. Restored.